### DISCUSSION TITLE
Welcome to novathesis Discussions!

### DISCUSSION THREAD

**Discussion by joaomlourenco** [joaomlourenco/novathesis#142] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MirandaVitor** [comment]
Hi,

I am using this template, however I noticed that the bibliography is not following the IEE style correctly (https://pitt.libguides.com/citationhelp/ieee). Independently of what kind of document I am citing, It  always shows up in the bibliography in the citing style for a "Chapter in book". Take as an example the citation it gives for an article: "D. Powers and W. Whaling.  “Range of Heavy Ions in Solids.” In:Physical Review 126.1 (1962), pp. 61–69.", when it should be something like this : "D. Powers and W. Whaling.  “Range of Heavy Ions in Solids.”, Physical Review, vol. 126, no. 1, pp. 695-701, 1962.". If someone knows how to solve this, I would be thankful.

Cheers,
Carlos Vítor

**Reply by joaomlourenco** [reply]
Hi Vitor,

Which version of the template are you using?  
Which options did you use to select the  bibliography style?

**Reply by MirandaVitor** [reply, marked as answer]
Hi Professor João Lourenço,

I checked and I was using the default citing style that is not IEEE nor APA.
I changed it and now is fine.

Thank you.
